politeiawww/api/www/v1/v1.go

```diff
@@ -285,6 +287,14 @@ const (
 	NotificationEmailAdminProposalVoteAuthorized EmailNotificationT = 1 << 6
 	NotificationEmailCommentOnMyProposal         EmailNotificationT = 1 << 7
 	NotificationEmailCommentOnMyComment          EmailNotificationT = 1 << 8
+
+	// Category types
```

Comment:
How were these defined?  Do they match the cms invoice categories?  If so, explain that in the comment.  Non-obvious things should be explicitly documented.

Reply:
done :+1: